data/testsuite

```diff
@@ -498,3 +498,229 @@ dalla bat kææwa
+
+# 4 Cardinal numeral adjective, cardinal 'one' preceding noun
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Adjectives}
+mam2 ek2 pot2 gatta
+ma-m2 ek2 pot2 gatta
+I-NOM one book.SG buy.PST
+`I bought one book`
+
+# 5 Cardinal numeral adjective, incorrect cardinal 'one' location
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Adjectives}
+mam2 pot2 ek2 gatta
+ma-m2 pot2 ek2 gatta
+I-NOM book.SG one buy.PST
+`I bought book one`
+
+# 6  Indefinite Adjective
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Adjectives}
+sam2har2 guruw2ru kææwa
+sam2har2 guruw2ru kææwa
+some teachers.PL eat.PST
+`Some teachers eat`
+
+# 7 Indefinite Adjective, incorrect - adjective follows noun
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Adjectives}
+guruw2ru sam2har2 bat kææwa
+guruw2ru sam2har2 bat kææwa
+teachers.PL some rice eat.PST
+`Teachers some ate rice`
+
+# 8 Cardinal Numeral Adjective, cardinal noun follows adjective
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Adjective}
+mam2 gas dek2 gatta
+ma-m2 gas dek2 gatta
+I-NOM trees.PL two buy.PST
+I bought 2 trees
+
+# 9 Cardinal Numeral Adjective, incorrect cardinal order, cardinal noun precedes
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Adjective}
+mam2 dek2 gas gatta
+ma-m2 dek2 gas gatta
+I-NOM two trees.PL buy.PST
+I bought 2 trees
+
+### Non-Verbal Predicates
+
+# 1 Dative nominal predicate
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Non-Verbal Predicates}
+dalla oyaa12
+dalla oyaa-12
+dog you-DAT
+`The dog is for you`
+
+# 2 Dative nominal predicate, missing dative marker
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Non-Verbal Predicates}
+dalla oyaa
+dalla oyaa
+dog you
+`The dog is you`
+
+# 3 Adjectival Predicate, with vowel-final adjective with assertion marker
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Non-Verbal Predicates}
+bat rasai
+bat rasa-i
+rice delicious-AM
+`The rice is delicious`
+
+# 4 Adjectival Predicate, vowel-final adjective, missing assertion marker
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Non-Verbal Predicates}
+bat rasa
+bat rasa
+rice delicious
+`The rice is delicious`
+
+# 5 Adjectival Predicate, consonant-final adjective
+Source: author
+Vetted: s
+Judgement: g
+Phenomena: {Non-Verbal Predicates}
+pot2 alut
+pot2 alut
+book.SG new
+`The book is new`
+
+# 6 Adjectival Predicate, vowel-final adjective with assertion marker
+Source: author
+Vetted: s
+Judgement: g
+Phenomena: {Non-Verbal Predicates}
+pot2 lokui
+pot2 loku-i
+book.SG big-AM
+`The book is big`
+
+# 7 Adjectival Predicate, missing vowel-final assertion marker
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Non-Verbal Predicates}
+pot2 loku
+pot2 loku
+book.SG big
+`The book is big`
+
+# 8 Nominal Predicate
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Non-Verbal Predicates}
+eyaa guruw2r2ek
+eyaa guruw2r2-ek
+She/he teacher-INDF
+`She is a teacher`
+
+# 9 Nominal Predicate, missing indefinite marker on noun
```

Comment:
Asking just cuz I didn't read this part, but does this not equate to the sentence "She is THE teacher"?